manifests/0000_26_cloud-controller-manager-operator_11_deployment.yaml

```
@@ -46,10 +46,10 @@ spec:
           --leader-elect-retry-period=26s \
           --leader-elect-resource-namespace=openshift-cloud-controller-manager-operator \
           "--images-json=/etc/cloud-controller-manager-config/images.json" \
-          --metrics-bind-address=:9258 \
+          --metrics-bind-address=:9257 \
```

This needs to be bound on the loopback only
```suggestion
          --metrics-bind-address=127.0.0.1:9257 \
```